### ISSUE TITLE
Visual Accessibility in EECOM/Eng

### ISSUE THREAD

**Issue by space-bass** [OCESS/orbitx#121]
'James Cooper was the Habitat EECOM for the 2006-07 Mission to Borrelly. Unfortunately, the crew of that mission had several close encounters with abnormal pressures due to Mr. Cooper being colour-blind, and therefore unable to distinguish between the green "safe" and yellow "warning" labels the software placed on the monitor. Prior to the mission, Mr. Cooper had not realized that there was more than one non-red color. Software rewrites were planned after this to avoid further disaster, but the need was gone after Mr. Cooper graduated from Lisgar. ' -OCESS wiki, 'EECOM' article

The time is now, my friends. No longer must our colourblind comrades struggle to use OCESS software. The solution is as simple as providing a little graphic alongside the colour, allowing everyone to have the quick at-a-glance reference without being inaccessible. Attached is a little mockup of what EECOM would look like with this new design.
![2020_12_01_0yv_Kleki](https://user-images.githubusercontent.com/67240009/100818409-13bdd700-3418-11eb-8568-1f9d76dd0047.png)

A similar design scheme can hopefully be adopted in Engineering as well.

**Comment by pmelanson** [comment]
Agreed. We haven't focused on EECOM's interface yet, but when we do so in the future we should definitely follow modern UI guidelines for colour.
